### ISSUE TITLE
Feature: Allow admin to validate the climber on D-day

### ISSUE THREAD

**Issue by scorreia** [dmossakowski/skala3ma#54]
On the day of the competition, the organizer will usually check that each climber is subscribed. 
It would be useful to have a checkbox for that in the administration panel. 
The checkbox would indicate whether the climber is present or absent on the day of the competition. 

Having this information will help to remove automatically all absent climbers from the results computation.

**Comment by dmossakowski** [comment]
Before final results, it's possible to review all climbers and remove any that have 0 routes climbed. Not sure about the rules but having a present climber for a gym who did not climb any routes probably shouldn't count for the club anyway.

(There is a possibility that a climber is present initially but then needs to leave before the start.)

**Comment by scorreia** [comment]
Yes, but the organizer nevertheless needs to welcome each participant and ensure that they are registered. I've seen that they usually use an Excel file for that. 
Having this feature in Skala3ma will avoid that back and forth between the app and the excel file.

**Comment by dmossakowski** [comment]
This implemented as of 20251114 through PR: https://github.com/dmossakowski/skala3ma/pull/122

There is a new display for climbers list and a climber can be checked-in using the same screen as later when adding routes

<img width="718" height="470" alt="Image" src="https://github.com/user-attachments/assets/692f87f4-3bca-4508-9d80-c7fc4bc03fec" />
